agent/src/main.cpp

```diff
@@ -177,9 +188,13 @@ int main(int argc, char **argv)
      * over the static configuration.
      */
     std::map<std::string, std::string> stringOptions;
-    if (servicemanifest != "") {
+    if (manifestDir != "") {
```

Comment:
serviceManifestDir or manifestDir?
If I use the first option and re-use the same for default it will be a very long variable name.